src/UI/BuildingMenu.c

```diff
@@ -254,8 +254,6 @@ static void buttonMenuItem(int item)
 			case BUILDING_FORT:
 				menu.selectedSubmenu = 24;
 				break;
-            default:

```

Comment:
GCC 4.9 actually complains about switches on enums that don't include a default case, that's why it's here.